firefox-ios/firefox-ios-tests/Tests/XCUITests/ExperimentIntegrationTests.swift

```diff
@@ -11,7 +11,7 @@ final class ExperimentIntegrationTests: BaseTestCase {
     override func setUpApp() {
         app.activate()
         let closeButton = app.buttons["CloseButton"]
-        if closeButton.waitForExistence(timeout: TIMEOUT_LONG) {
+        if closeButton.waitForExistence(timeout: TIMEOUT) {
```

Comment:
If you don't the onboarding screen, we may use a `LaunchArgument` before `app.activate` to prevent the screen from showing up.

Reply:
We can't use `LaunchArgument` because I am starting the app with `nimbus-cli` to enroll it into an experiment. I think we are passing arguments when we launch it via `nimbus-cli` but this is there just to make sure the onboard ui doesn't block the test.